src/ports/postgres/modules/deep_learning/test/madlib_keras_cifar.setup.sql_in

```diff
@@ -46,12 +46,12 @@ CREATE TABLE cifar_10_sample_text_batched AS
 
 -- Insert a new row with NULL as the dependent var (one-hot encoded)
 UPDATE cifar_10_sample_text_batched
-	SET dependent_var = convert_array_to_bytea(ARRAY[0,0,1,0,0]::REAL[]) WHERE buffer_id=0;
+	SET dependent_var = convert_array_to_bytea(ARRAY[0,0,1,0,0]::smallint[]) WHERE buffer_id=0;
```

Comment:
This function requires REAL[] as input, doesn't it?  Seems like we need one for SMALLINT defined also... or does it automatically cast it from SMALLINT to REAL as it runs?

Reply:
`convert_array_to_bytea` is overloaded and accepts both real[] and smallint[] as arg types.